packages/core/src/__tests__/error.test.ts

```diff
@@ -11,7 +25,16 @@ jest.mock("react-native", () => ({
         params: object,
       ) => {
         mockLogHandledError(message, componentStack, params);
-        return true;
+        return Promise.resolve(true);
+      },
+      logUnhandledJSException: (
+        name: string,
+        message: string,
+        errorType: string,
+        stack: string,
+      ) => {
```

Comment:
and here it can be an inline return for simplicity

```
logUnhandledJSException: (
        name: string,
        message: string,
        errorType: string,
        stack: string,
      ) => mockLogUnhandledJSException(name, message, errorType, stack)
```